### ISSUE TITLE
Add ignoreAttribute option

### ISSUE THREAD

**Issue by 7ixi0** [edvardchen/eslint-plugin-i18next#6]
I am using this plugin with React and Typescript and I have found it very useful so far.
However, I think it would be useful to add an `ignoreAttribute` option, similar to `ignoreCallee` but for JSX attributes.

The reason is:
```jsx
<TextField
  variant="outlined"
  margin="normal"
  name="newEmail"
  autoComplete="email"
/>
```
> [TextField](https://material-ui.com/components/text-fields/) from material-ui

In the example above `variant` and `margin` are not reported as errors because their types are defined as a an union type of different strings.
But `name` and `autocomplete` are of type `string` so they get reported.

An `ignoreAttribute` option is necessary because the other solution would be spamming every file with `eslint-disable-next-line` comments.
And adding other attributes to the whitelist (like has been done with #4) for every framework and library that comes along would just be unfeasible.

I could work on the PR myself, just need to know if you would merge it.

**Comment by edvardchen** [comment]
It's great. I think there are some cases that UPPERCASE and union type of strings cannot handle in an instinctive way.

Expect your PR.
